tests/fixtures/token_factory.py

```diff
@@ -21,8 +21,12 @@ class TokenNFTFactory(Factory):
     class Meta:
         model = TokenNFT
 
-    type = lazy_attribute(lambda o: fake.random_element(list(TokenNFTType)))
-    file = lazy_attribute(lambda o: f"http://{fake.domain_name()}{fake.file_path(category=o.type.value.lower())}")
```

Comment:
```python
file = lazy_attribute(lambda o: f"http://{fake.domain_name()}{fake.file_path(category=o.type.value.lower(), extension='pdf' if o.type.value.lower() == 'pdf' else None)}")
```

The lib used to generate will ignore `category` if `extension` is passed, this would fix the problem and not alter the factory logic.

Reply:
Can you see again please?